server/datastore/mysql/migrations/tables/20220704083846_AddSoftwareIDToSoftwareCVE.go

```diff
@@ -0,0 +1,30 @@
+package tables
```

Comment:
Because [this](https://github.com/fleetdm/fleet/pull/6489/files#diff-b3bc0cddb27020c420f2cd1a3996e9d9b9a39bc2f8c1799f8a3671e2f29609d9R31) will fail if the vulnerability cron job is also trying to delete one of the rows being updated (while unlikely is a possible scenario) - so we want to be able to re-run the second migration as many times as needed.

Reply:
I believe you could use [columnExists](https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/migrations/tables/migration.go#L14) to make the whole thing idempotent.